app/hooks/use-account-limits.ts

```diff
@@ -0,0 +1,108 @@
+import * as React from "react"
+
+import { ApolloError, ApolloQueryResult, gql, useQuery } from "@apollo/client"
+import crashlytics from "@react-native-firebase/crashlytics"
+
+const ACCOUNT_LIMITS_QUERY = gql`
```

Comment:
> Can you move this into galoy-client?

Not at the moment. Till the limits PR on the backend is merged. In my opinion, I'd suggest we hold off adding it to the galoy-client until we finalise the KYC mutations and query to avoid continuously making changes to the galoy client code